File: 0062-unique-paths/0062-unique-paths.php
class Solution {

    /**
     * @param Integer $m
     * @param Integer $n
     * @return Integer
     */
    function uniquePaths($m, $n) {
        $grid = [];
        for ($i = 0; $i < $m; $i++) {
            for ($j = 0; $j < $n; $j++) {
                if ($i == 0 || $j == 0) {
                    $grid[$j][$i] = 1;
                } else {
                    $grid[$j][$i] = $grid[$j][$i - 1] + $grid[$j - 1][$i];
                }
            }
        }
        return $grid[$n - 1][$m - 1];
    }
}

File: 0062-unique-paths/0062-unique-paths.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        grid = [[1 for _ in range(n)] for _ in range(m)]
            
        i = 1
        
        while i < m:
            j = 1
            while j < n:
                grid[i][j] = grid[i-1][j] + grid[i][j-1]
                j += 1
            i += 1
        
        return grid[m-1][n-1]
        